README.md

```diff
@@ -177,4 +177,16 @@ terraform apply
 
 The -auto-approve flag has been set on the pipeline to avoid having to manually approve the plan prior to infrastructure creation/change.
 
-The Main.tf file configures the backend to save the terraform state file on Azure blob storage.  If the file was stored with the main repo program secrets would not be securely encrypted.  By using Blob the file is encrypted and easily accessed by our pipeline.
\ No newline at end of file
+The Main.tf file configures the backend to save the terraform state file on Azure blob storage.  If the file was stored with the main repo program secrets would not be securely encrypted.  By using Blob the file is encrypted and easily accessed by our pipeline.
+
+## lOGGING
+
+The app and utilizes Loggly to to compile application logs and make them easier to search.
+
+```bash
+https://www.loggly.com/signup/
+```
+
+In order for this to work an environment vairable called LOGGLY_TOKEN needs to be added with your personal Loggly token.
```

Comment:
Just a minor spelling error - `variable` rather than `vairable`, and similarly in the sentence after
```suggestion
In order for this to work an environment variable called LOGGLY_TOKEN needs to be added with your personal Loggly token.
```